### ISSUE TITLE
Unable to leave private repo room for which I'm no longer authorised.

### ISSUE THREAD

**Issue by nmcb** [gitterHQ/gitter#808]
I'm unable to leave (remove from gitter's left pane room overview) the [anwb-dvb](https://gitter.im/anwbhv/anwb-dvb) repository room for which I'm no longer authorised. The `leave` button appears but clicking it has no effect. The issue is minor, just visually annoying.

**Comment by MadLittleMods** [comment]
Hey @nmcb,

 - What do you mean by "no longer authorized"?
 - What happens if you hide the room?
    - ![](http://i.imgur.com/z850h2a.png)

**Comment by nmcb** [comment]
- That I'm no longer a github registered collaborator for this repo.
- Just like the leave button, clicking hide has no effect.
- Additional, clicking the room link shows a "404, this is not the room..."

**Comment by Garbee** [comment]
Just checked network. Getting a 403 forbidden error when trying to hide a room I no longer have access to.

**Comment by mydigitalself** [comment]
Yup, normally it should automatically eject you from the room when you get that, some changed to room membership must have had an undesired side-effect.

**Comment by Garbee** [comment]
Ah, membership change would make sense. That is what happened in the case I am seeing.

**Comment by kbrock** [comment]
Think this is similar, if it is different, let me know and I'll create another issue.

1. Join a private organization room. (other private rooms may work as well, not sure)
2. Leave private room
3. 25 of your "closest friend" join the private room taking it to max capacity.
4. Have a "close friend" notify you in that channel via `@user` (I guess, I don't know because I can't get in)
5. Try and close room or hide notification... error
6. Enjoy Gitter app always telling you that you have outstanding messages.

**Comment by nmcb** [comment]
@kbrock I believe it is a separate issue as this one seems to be specific to not being able to leave a room which one is not authorised to access anymore.

**Comment by Garbee** [comment]
What @kbrock is  seeing is the same issue as this. The problem is they no longer have access to the repo, so they can't leave it nor see any mentions. Yet, it will always show they have unread messages.

**Comment by thisismygithub** [comment]
Similar problem, 
1. I created a new private channel and did not add anyone
2. I clicked leave button 
3. Additional, clicking the room link shows a "404, this is not the room..."

**Comment by MadLittleMods** [comment]
@thisismygithub This is not the same issue.You can track https://github.com/gitterHQ/gitter/issues/1193 instead.
